Dockerfile

```diff
@@ -1,7 +1,18 @@
-ARG BUILD_ARCH=amd64
-FROM ${BUILD_ARCH}/debian:buster-slim
+FROM python:3.7-alpine
```

Comment:
use alpine instead of debian since the image is only 37mb big vs 64mb
However since they use different implementations of libc they might perform different in terms of the runtime. Most likely the difference is not big enough for us to care. Here is a comparision: http://www.etalabs.net/compare_libcs.html